core/src/Kokkos_View.hpp

```diff
@@ -1995,26 +1995,24 @@ class View : public ViewTraits< DataType , Properties ... > {
    typedef typename traits::device_type::memory_space specialized_memory_space;
    if ( Kokkos::Impl::is_resilient_space< specialized_memory_space >::value &&
         Kokkos::Impl::SharedAllocationRecord<void,void>::duplicates_enabled() ) {
-        printf("duplicating shared alloc record.. \n");
       shared_record_type  *
          record = reinterpret_cast<shared_record_type*>(m_map.duplicate_shared( m_track.template get_record<specialized_memory_space>() ));
 
        // Setup and initialization complete, start tracking
        m_track.assign_allocated_record_to_uninitialized( record );
-       printf("after duplicate record assigned to tracker: %d \n", m_track.tracking_disabled());
      }
 #endif
 
   }
 
   KOKKOS_INLINE_FUNCTION
```

Comment:
You should be able to `default` all of these copy and move constructors and copy and move assignment operators.